server/filestore.go

```diff
@@ -173,7 +173,7 @@ const (
 	// cache idx expiration
 	defaultCacheIdxExpiration = 5 * time.Minute
 	// default sync interval
-	defaultSyncInterval = 10 * time.Second
+	defaultSyncInterval = 60 * time.Second
```

Comment:
If sync has really a negative impact, this is just moving the issue from 10sec to 60sec.
Wonder if you should not expose the (auto)sync params so users can decide.

Reply:
Its is configurable but not all the way to user at this time.